crates/youki/src/commands/completion.rs

```diff
@@ -0,0 +1,15 @@
+use clap::{App, Parser};
+use clap_generate::{generate, Generator, Shell};
+use std::io;
+
+#[derive(Debug, Parser)]
```

Comment:
youki -h currently does not have a description for this command

```suggestion
Generate scripts for shell completion
#[derive(Debug, Parser)]
```